linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/oval/shared.xml

```diff
@@ -40,19 +40,11 @@
   comment="Tests the value of the ClientAliveCountMax setting in the /etc/ssh/sshd_config file"
   id="test_sshd_clientalivecountmax" version="1">
     <ind:object object_ref="obj_sshd_clientalivecountmax" />
-    <ind:state state_ref="state_sshd_clientalivecountmax" />
```

Comment:
You can just use the jinja macros in this situation where xccdf variables are not needed anymore in the OVAL check. Something like:

`{{{ oval_sshd_config(parameter="ClientAliveCountMax", value="0" }}}`

Reply:
Actually you could even use the template for this rule in the `rule.yml`:

```
template:
    name: sshd_lineinfile
    vars:
        parameter: ClientAliveCountMax
        rule_id: sshd_set_keepalive
        value: '0'
```